.env.template

```diff
@@ -6,5 +6,5 @@ DB_NAME=
 DB_USER=
 DB_PASS=
 JWT_EXPIRES_IN_SECONDS=3600
-JWT_PUBLIC_KEY_PATH=
-JWT_PRIVATE_KEY_PATH=
+JWT_PUBLIC_KEY=
```

Comment:
Shall we rename these vars to `JWT_PUBLIC_KEY_BASE64` for less ambiguity? :thinking:

Reply:
Yes definitely